common/src/main/java/com/graphicsfuzz/common/util/UpgradeShadingLanguageVersion.java

```diff
@@ -200,13 +270,22 @@ public void visitVariablesDeclaration(VariablesDeclaration variablesDeclaration)
     */
     visit(baseType);
     for (VariableDeclInfo vdi : variablesDeclaration.getDeclInfos()) {
+      if ((flags & NORENAME) == 0) {
+        // To avoid collisions with builtins, add underscore to variable names
+        final String newname = vdi.getName() + "_";
+        // Rename any occurrences of the variable name too
+        variableRename.put(vdi.getName(), newname);
+        vdi.setName(newname);
+      }
       if (vdi.hasArrayInfo()) {
         visit(vdi.getArrayInfo().getSizeExpr());
       } else if (baseType instanceof ArrayType) {
         visit(((ArrayType) baseType).getArrayInfo().getSizeExpr());
       }
       if (vdi.hasInitializer()) {
         visit(vdi.getInitializer());
```

Comment:
Please format this as:

// TODO(https://github.com/google/graphicsfuzz/issues/842) Only rename if ...

i.e., put the URL in the parentheses.

Reply:
Done. Also, this comment was in a wrong place, moved it to the right one.